spec/routing/search_terms_routing_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe SearchTermsController, type: :routing do
+  describe "routing" do
+    it "routes to #index" do
+      expect(get: "/search_terms").to route_to("search_terms#index")
+    end
+
+    it "routes to #new" do
+      expect(get: "/search_terms/new").to route_to("search_terms#new")
+    end
+
+    it "routes to #show" do
+      expect(get: "/search_terms/1").to route_to("search_terms#show", id: "1")
+    end
+
+    it "routes to #edit" do
+      expect(get: "/search_terms/1/edit").to route_to("search_terms#edit", id: "1")
+    end
+
+
+    it "routes to #create" do
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/c919a05a12101ae07b4c21bf82ccf33697056e9e/spec/routing/search_terms_routing_spec.rb#L22"></span>Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>